src/BoilerPlate/App.cpp

```diff
@@ -3,10 +3,11 @@
 #include <algorithm>
 #include "ColorRepresentation.h"
 // OpenGL includes
-#include <SDL2/SDL_opengl.h>
+#include <SDL_opengl.h>
```

Comment:
I'd move all of SDL dependencies to a file called `IncludeSDL.hpp`. I'd be easier to update in case something else needs to be added since I'd be in a single place rather than a lot of source code files.